### PR TITLE
Update kbdfans/kbd75/digital

### PR DESCRIPTION
**Pull request by mikebell**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd75/keymaps/digital/keymap.c [hunk 1]
```diff
@@ -1,8 +1,11 @@
 #include QMK_KEYBOARD_H
 
+#define DEF_LAYER 0
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/keymaps/digital/keymap.c]: You can potentially use an enum here

```c
enum my_layers {
    DEF_LAYER,
    FN_LAYER
};
```

File: keyboards/kbdfans/kbd75/keymaps/digital/config.h [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/* turn off RGB when computer sleeps */
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/keymaps/digital/config.h]:
```suggestion
#pragma once

/* turn off RGB when computer sleeps */
```

File: keyboards/kbdfans/kbd75/keymaps/digital/keymap.c [hunk 3]
```diff
@@ -11,12 +14,29 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
+    _______,  KC_LEFT,  KC_DOWN,  KC_RGHT,  KC_PGDN,  _______,  _______,  _______,  _______,  _______,  _______,  _______,                      _______,  _______,
     _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,  _______,
     _______,  _______,  _______,                      _______,  _______,  _______,                      BL_TOGG,  BL_INC,   BL_DEC,   _______,  _______,  _______
   )
 };
+
+uint32_t layer_state_set_user(uint32_t state) {
+    switch (biton32(state)) {
+    case FN_LAYER:
+        rgblight_setrgb_white();
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/keymaps/digital/keymap.c]:
```suggestion
        rgblight_sethsv_noeeprom_white();
```

File: keyboards/kbdfans/kbd75/keymaps/digital/keymap.c [hunk 4]
```diff
@@ -11,12 +14,29 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
+        break;
+    default:
+        rgblight_setrgb_cyan();
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/keymaps/digital/keymap.c]:
```suggestion
        rgblight_sethsv_noeeprom_cyan();
```

File: keyboards/kbdfans/kbd75/keymaps/digital/keymap.c [hunk 5]
```diff
@@ -11,12 +14,29 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
+        break;
+    }
+    return state;
+}
+
+// Runs just one time when the keyboard initializes.
+void keyboard_post_init_user(void) {
+  rgblight_setrgb_cyan();
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/keymaps/digital/keymap.c]:
```suggestion
  rgblight_sethsv_noeeprom_cyan();
```

File: keyboards/ymdk/ymd09/keymaps/digital/keymap.c [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+               RGB_HUD, RGB_VAD, RESET),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QUITZOOM:
+      if (record->event.pressed) {
+        SEND_STRING(SS_LALT("q"));
+        _delay_ms(1000);
```
Comment (fauxpark) [inline keyboards/ymdk/ymd09/keymaps/digital/keymap.c]:
```suggestion
        wait_ms(1000);
```

File: keyboards/ymdk/ymd09/keymaps/digital/keymap.c [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+               RGB_HUI, KC_TRNS, RGB_SAI,
+               RGB_HUD, RGB_VAD, RESET),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QUITZOOM:
+      if (record->event.pressed) {
+        SEND_STRING(SS_LALT("q"));
```
Comment (fauxpark) [inline keyboards/ymdk/ymd09/keymaps/digital/keymap.c]:
```suggestion
        tap_code16(A(KC_Q));
```

File: keyboards/ymdk/ymd09/keymaps/digital/keymap.c [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+        _delay_ms(1000);
+        register_code(KC_ENT);
```
Comment (fauxpark) [inline keyboards/ymdk/ymd09/keymaps/digital/keymap.c]:
```suggestion
        tap_code(KC_ENT);
```
I think you probably meant to do this instead, as `register_code()` only sends the keydown and I don't see where the corresponding `unregister_code()` is sent.

**Pull request closed without merging**